docs/patterns/existing-eks-observability-accelerators/existing-eks-opensource-observability.md

```diff
@@ -12,6 +12,7 @@ please read the [service documentation](https://docs.aws.amazon.com/eks/latest/u
 ## Objective
 
 Configure the existing Amazon EKS cluster with below Observability components;
+- Control Plane logging
```

Comment:
This change is not required. I would rather add a bullet to say. If Control Plane logging is not enabled.. please use enableControlPlane logging function attaching a small relevant code snippet.